docs/intro/setup.rst

```diff
@@ -72,29 +72,35 @@ Here we give line-by-line instructions to demonstrate the general workflow.
 .. code-block:: python
 
    import hls4ml
+   import tensorflow as tf
+   from tensorflow.keras.layers import Dense
 
-   # Fetch a keras model from our example repository
-   # This will download our example model to your working directory and return an example configuration file
-   config = hls4ml.utils.fetch_example_model('KERAS_3layer.json')
+   # Construct a basic, untrained keras model
+   model = tf.keras.models.Sequential()
+   model.add(Dense(64, input_shape=(1,), name='Dense', kernel_initializer='lecun_uniform', kernel_regularizer=None))
 
-   # You can print it to see some default parameters
-   print(config)
+   # Generate an hls configuration from the keras model
+   config = hls4ml.utils.config_from_keras_model(model)
 
-   # Convert it to a hls project
-   hls_model = hls4ml.converters.keras_to_hls(config)
+   # You can print the config to see some default parameters
+   print(config)
 
-   # Print full list of example model if you want to explore more
-   hls4ml.utils.fetch_example_list()
+   # Convert the model to an hls project using the config
+   hls_model = hls4ml.converters.convert_from_keras_model(
+      model=model,
+      hls_config=config,
+      backend='Vitis'
+   )
 
```

Comment:
We need to add the compile step. Just to make it a bit more realistic, it might be useful to add more input parameters, another layer and an activation function. It would be only like 2 more lines of code.